src/functions/env.ts

```diff
@@ -123,7 +123,11 @@ export function parse(data: string): ParseResult {
   return { envs, errors };
 }
 
-class KeyValidationError extends Error {}
+export class KeyValidationError extends Error {
+  constructor(public key: string, public message: string) {
+    super();
```

Comment:
Pass an actual message to `super()` so that a future developer can actually see what's going on with `err.message`